third_party/2and3/yaml/__init__.pyi

```diff
@@ -14,10 +14,10 @@ def scan(stream, Loader=...): ...
 def parse(stream, Loader=...): ...
 def compose(stream, Loader=...): ...
 def compose_all(stream, Loader=...): ...
-def load(stream, Loader=...): ...
-def load_all(stream, Loader=...): ...
-def safe_load(stream): ...
-def safe_load_all(stream): ...
+def load(stream: Union[str, IO[str]], Loader: Loader=...) -> Any: ...
```

Comment:
I discovered a problem with the annotation for `Loader` here. IIUC it's intended that this argument can be an instance of `BaseLoader` or `SafeLoader` as well. But given the inheritance structure in `loader.pyi` that doesn't work -- if I pass an instance of `SafeLoader` I get a type error. Maybe you can fix this by declaring this argument (here and below) as `Union[BaseLoader, SafeLoader, Loader]`? (Do double check that I got this right -- I'm not a user of yaml myself, but I found this caused a regression in an internal Dropbox codebase that uses it.)

Reply:
I fixed this in #1695.